src/backend/metal/src/device.rs

```diff
@@ -6,17 +6,19 @@ use {conversions as conv, command, native as n};
 use native;
 
 use std::borrow::Borrow;
-use std::collections::hash_map::{Entry, HashMap};
+use std::collections::HashMap;
```

Comment:
why this change? it's the same as the old code but more verbose

Reply:
Yeah I need to combine these again. Originally `HashMap` was removed entirely then I realized we still need it for the spirv_cross overrides